### PR TITLE
upgrade base rpc

### PR DESCRIPTION
**Pull request by angel-ding-cb**
Added developer access rpc to base mainnet

### CODE REVIEW

File: _data/chains/eip155-8453.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Base",
   "chain": "ETH",
-  "rpc": ["https://mainnet.base.org/"],
+  "rpc": ["https://mainnet.base.org/", "https://developer-access-mainnet.base.org"],
```
Comment (anikaraghu) [inline _data/chains/eip155-8453.json:4]: nit: multiple lines and I think we need a slash at the end 
```suggestion
  "rpc": [
      "https://mainnet.base.org/",
       "https://developer-access-mainnet.base.org/"
    ],
```

**Comment by angel-ding-cb** [comment]
related PR: https://github.com/ethereum-lists/chains/pull/3247

**Review by anikaraghu — APPROVED** [review]

File: _data/chains/eip155-8453.json [hunk 2]
```diff
@@ -1,7 +1,10 @@
 {
   "name": "Base",
   "chain": "ETH",
-  "rpc": ["https://mainnet.base.org/"],
+  "rpc": [
+      "https://mainnet.base.org/",
+       "https://developer-access-mainnet.base.org/"
```
Comment (aflock) [inline _data/chains/eip155-8453.json:6]: small whitespace inconsistency?

**Review by Loloo651 — APPROVED** [review]

**Comment by ligi** [comment]
@wbnns can you comment on this? Was removed in your PR

**Comment by wbnns** [comment]
@ligi 

Looks good, thanks! The reason we are adding this back is because it caused a downstream issue with a provider.

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Review by ligi — APPROVED** [review]

**Pull request merged**